fastly/request.go

```diff
@@ -54,6 +55,7 @@ func (c *Client) RawRequest(verb, p string, ro *RequestOptions) (*http.Request,
 	// Set the API key.
 	if len(c.apiKey) > 0 {
 		request.Header.Set(APIKeyHeader, c.apiKey)
+		request.Header.Set("Authorization", fmt.Sprintf("Bearer %s", c.apiKey))
```

Comment:
Should this be `c.apiKey`? That would set the value of the bearer to the API token and not the desired OAuth JWT `access_token`,